### PR TITLE
[plugin.video.zdf_de_2016] 1.0.14

### PR DESCRIPTION
**Pull request by generia**
### Description
1.0.14 (2019-01-29)
- fix for missing play icon

### Checklist:

- [x] My code follows the [add-on rules](http://kodi.wiki/view/Add-on_rules) and [piracy stance](http://kodi.wiki/view/Official:Forum_rules#Piracy_Policy) of this project. 
- [x] I have read the [CONTRIBUTING](https://github.com/xbmc/repo-plugins/blob/master/CONTRIBUTING.md) document
- [x] Each add-on submission should be a single commit with using the following style: [plugin.video.foo] v1.0.0

Additional information :
- Submitting your add-on to this specific branch makes it available to any Kodi version equal or higher than the branch name with the applicable Kodi dependencies limits.
- [add-on development](http://kodi.wiki/view/Add-on_development) wiki page.
- Kodi [pydocs](http://kodi.wiki/view/PyDocs) provide information about the Python API
- [PEP8](https://www.python.org/dev/peps/pep-0008/) codingstyle which is considered best practice but not mandatory.
- This add-on repository has automated code guideline check which could help you improve your coding. You can find the results of these check at [Codacy](https://www.codacy.com/app/Kodi/repo-plugins/dashboard). You can create your own account as well to continuously monitor your python coding before submitting to repo.
- Development questions can be asked in the [add-on development](http://forum.kodi.tv/forumdisplay.php?fid=26) section on the Kodi forum.

### CODE REVIEW

File: plugin.video.zdf_de_2016/addon.xml [hunk 1]
```diff
@@ -1,5 +1,5 @@
 <?xml version="1.0" encoding="UTF-8" standalone="yes"?>
-<addon id="plugin.video.zdf_de_2016" name="ZDF Mediathek 2016" version="1.0.13" provider-name="Generia">
+<addon id="plugin.video.zdf_de_2016" name="ZDF Mediathek 2016" version="1.0.14" provider-name="Generia">
   <requires>
     <import addon="xbmc.python" version="2.14.0"/>
```
Comment (Rechi) [inline plugin.video.zdf_de_2016/addon.xml]: > `WARN: For jarvis it is advised to set xbmc.python version to 2.24.0`
```suggestion
     <import addon="xbmc.python" version="2.24.0"/>
```

**Comment by Rechi** [comment]
@generia please update the following except syntax changes.
```
--- a/plugin.video.zdf_de_2016/de/generia/kodi/plugin/frontend/zdf/player/TokenCache.py
+++ b/plugin.video.zdf_de_2016/de/generia/kodi/plugin/frontend/zdf/player/TokenCache.py
@@ -20,7 +20,7 @@
                     apiToken = line.strip()
                     self.log.info(LOG_PREFIX + "loading api-token '{}' from store-file '{}'", apiToken, self.storeFile)
                     break
-        except IOError, e:
+        except IOError as e:
             self.log.error(LOG_PREFIX + "caught exception while loading api-token using store-file '{}', exception: {}", self.storeFile, e)
             
         return apiToken
@@ -32,7 +32,7 @@
             if apiToken is not None:
                 file.write(apiToken)
             file.close()
-        except IOError, e:
+        except IOError as e:
             self.log.error(LOG_PREFIX + "caught exception while saving api-token using store-file '{}', exception: {}", self.storeFile, e)
 
     def _checkStorePath(self):
@@ -41,6 +41,6 @@
             return
         try:
             os.makedirs(storeDir)
-        except IOError, e:
+        except IOError as e:
             self.log.error(LOG_PREFIX + "caught exception while creating store-dir '{}', exception: {}", storeDir, e)
             return
--- a/plugin.video.zdf_de_2016/de/generia/kodi/plugin/frontend/zdf/search/SearchHistory.py
+++ b/plugin.video.zdf_de_2016/de/generia/kodi/plugin/frontend/zdf/search/SearchHistory.py
@@ -61,7 +61,7 @@
                     entry = self._parseEntry(line)
                     if entry is not None:
                         entries.append(entry)
-        except IOError, e:
+        except IOError as e:
             self.log.error(LOG_PREFIX + "caught exception while loading search history using store-file '{}', exception: {}", self.storeFile, e)
         return entries
                 
@@ -91,7 +91,7 @@
                 date = entry.date #getDateFormatted()
                 file.write('{0} {1} {2}\n'.format(date, entry.contentTypes, entry.query))
             file.close()
-        except IOError, e:
+        except IOError as e:
             self.log.error(LOG_PREFIX + "caught exception while saving search history using store-file '{}', exception: {}", self.storeFile, e)
 
     def _checkStorePath(self):
@@ -100,7 +100,7 @@
             return
         try:
             os.makedirs(storeDir)
-        except IOError, e:
+        except IOError as e:
             self.log.error(LOG_PREFIX + "caught exception while creating store-dir '{}', exception: {}", storeDir, e)
             return
         
```

**Comment by Rechi** [comment]
> Each add-on submission should be a single commit with using the following style: [plugin.video.foo] v1.0.0

@generia please squash your commits.

**Comment by Rechi** [comment]
@generia I still see two commits. If you need instructions just ask.

**Comment by generia** [comment]
forgot to finish the rebase. squash is done now.
Thanks for the review and you patience :)

**Pull request merged**